hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/distributions/state/ManageDistUIState.java

```diff
@@ -48,14 +45,14 @@
 
     private Long lastSelectedDistribution;
 
-    private Set<Long> selectedSoftwareModules = emptySet();
+    private Long lastSelectedSoftwareModule;
```

Comment:
Better change the type to `Optional<Long>` so if I call the getter mutliple times in a row you do not create a optional wrapper all the time and it is clear that the instance variable can be null.